### PR TITLE
Update linux-instructions.md

### PR DESCRIPTION
**Pull request by lasombra**
Added Fedora 23 required packages

### CODE REVIEW

File: Documentation/building/linux-instructions.md [hunk 1]
```diff
@@ -43,6 +43,9 @@ Then install the packages you need:
 
 You now have all the required components.
 
+For Fedora 23 you will need to install the following:
+`$ sudo dnf cmake install clang lldb-devel libunwind-devel lttng-ust-devel libuuid-devel`
+
```
Comment (gkhanna79) [inline Documentation/building/linux-instructions.md]: Can you please rephrase the section content to layout the commands for various platforms (Linux, Fedora, etc) in a tabular manner?

**Comment by godu** [comment]
`libicu-devel` missing ?

**Comment by Petermarcu** [comment]
@gkhanna79 , next steps here? Does this whole doc need cleanup now that we're a year later?

**Comment by gkhanna79** [comment]
@Petermarcu Waiting for response to the feedback.

**Comment by Petermarcu** [comment]
Its been almost a year since the first PR. @lasombra , did you want to finish this? We'll assume not if we don't hear back in a week. If not, we'll close the PR in aand @gkhanna79 you can decide what updates we need to the instructions.

**Comment by lasombra** [comment]
This completely bypassed my radar for some reason. I'm sorry it took so
long. I will take care of it until the end of the week.

**Comment by Petermarcu** [comment]
@lasombra, thanks!

**Comment by lasombra** [comment]
@Petermarcu That should do it

**Pull request merged**